### ISSUE TITLE
Updated rule: Image filename is accessible name for image

### ISSUE THREAD

**Issue by Jym77** [w3c/wcag-act#480]
The ACT-Rules community group has completed work on "Image filename is accessible name for image". We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: Image filename is accessible name for image
**Rule id**: 9eb3f6
**Link to the rule**: https://act-rules.github.io/rules/9eb3f6
**License**: W3C Software And Document Notice And License
**Implementors**:
- QualWeb
- SortSite

If you'd like to contact me, please e-mail me at jym@siteimprove.com. (optional: You can also reach me on the W3C Community Slack, my name there is Jean-Yves Moyen)

**Comment by maryjom** [comment]
[Survey results](https://www.w3.org/2002/09/wbs/93339/ACTImageFileName/results) for the survey due on 24 September.

**Comment by maryjom** [comment]
Discussion in [24 September meeting](https://www.w3.org/2020/09/24-wcag-act-minutes.html#ResolutionSummary).  [Resolution](https://www.w3.org/2020/09/24-wcag-act-minutes.html#ResolutionSummary) made to discuss with the author to take the rule to AG. @WilcoFiers Is there anything else we need to do here? Does this mean it's waiting to publish?

**Comment by WilcoFiers** [comment]
Alrighty, closing this issue, the CG has decided it will deprecated the rule based on feedback we gave them. The PR for it is here: https://github.com/act-rules/act-rules.github.io/pull/1538
